frontend/src/App.vue

```diff
@@ -83,10 +103,17 @@ export default {
         };
       }
     },
+    changeItemsState(changedItems) {
+      changedItems.forEach(itement => {
+        const tarItem = this.getItemByFqid(itement.fqid);
+        if (!tarItem) {
```

Comment:
This block is never entered in the tests, so coverage is not full and 
the build is currently partially broken

```javascript
        if (!tarItem) {
          return;
        }
```